### PR TITLE
Test unforgiving dread knight

### PR DESCRIPTION
**Pull request by qwrtln**
The goal is to disallow dread knight shrinking, which allowed some overflows to pass page checks.

before|after
en-35

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Image please

**Comment by qwrtln** [comment]
> Image please

It's WIP, sorry

**Comment by qwrtln** [comment]
Verbose translations will no longer be able to bypass the checks.

File: sections/combat.tex [hunk 1]
```diff
@@ -204,12 +204,10 @@ \subsection*{Campaign and Cooperative Combat}
...
 
 \vspace*{\fill}
-\begin{scaledfigure}[blanker]
-  \centering
-  \includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/dread_knight.jpg}
-\end{scaledfigure}
 
-\clearpage
+\begin{center}
+  \includegraphics[width=0.97\linewidth]{\art/dread_knight.jpg}
```
Comment (tomaas-zeman) [inline sections/combat.tex:209]: Not a big deal but to my surprise, I was able to squeeze Czech translation to a point where the Dread knight can be at full width, which looks slightly better. Maybe we could do the same with other languages

Screenshot 2024-06-11 at 13 45 35

**Comment by qwrtln** [comment]
Agreed, but currently impossible without changing original text.

11 Jun 2024 13:46:48 Tomas Zeman ***@***.***>:

**Review by tomaas-zeman — APPROVED** [review]

**Comment by qwrtln** [comment]
I have no idea why, but German yields different results locally and in the workflow. Apart from the word "ends" carried over to the next column, everything else seems identical.

@DarthGandalf @tomaas-zeman 

image

**Comment by tomaas-zeman** [comment]
> I have no idea why, but German yields different results locally and in the workflow. Apart from the word "ends" carried over to the next column, everything else seems identical.

Isnt it because workflow has lua enabled and locally its disabled?

**Comment by qwrtln** [comment]
> > I have no idea why, but German yields different results locally and in the workflow. Apart from the word "ends" carried over to the next column, everything else seems identical.
> 
> Isnt it because workflow has lua enabled and locally its disabled?



> > I have no idea why, but German yields different results locally and in the workflow. Apart from the word "ends" carried over to the next column, everything else seems identical.
> 
> Isnt it because workflow has lua enabled and locally its disabled?

Could be. I changed the workflow locally but haven't pushed yet. You can see that lua performs hyphenation in different places.

**Comment by qwrtln** [comment]
po4a check

**Comment by qwrtln** [comment]
@DarthGandalf What do you think? It should prevent any future abuse.

**Review by DarthGandalf — APPROVED** [review]

**Comment by DarthGandalf** [comment]
I don't have a strong opinion on this.

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**